db/re/item_db_etc.yml

```diff
@@ -9769,7 +9769,7 @@ Body:
     Buy: 20
     Weight: 10
     Locations:
-      Right_Hand: true
+      Armor: true
```

Comment:
The locations depend on the server. I'm in favor of leaving it in the right hand like drake card